app/src/sharedTest/java/org/oppia/app/testing/DragDropTestActivityTest.kt

```diff
@@ -104,7 +105,10 @@ class DragDropTestActivityTest {
   }
 
   private fun createDragCallback(activity: DragDropTestActivity): ItemTouchHelper.Callback {
-    return DragAndDropItemFacilitator(activity as OnItemDragListener)
+    return DragAndDropItemFacilitator(
+      activity as OnItemDragListener,
+      activity as OnDragEndedListener
+    )
```

Comment:
line 113 is  Unexpected blank line(s) before "}"

Reply:
done